### PR TITLE
only send meaningful address values

### PR DESCRIPTION
**Pull request by sirLisko**
**What**  
Filtered out fake values inserted to make the DB happy.

*From:*
Screenshot 2020-06-20 at 09 26 12

*To:*
Screenshot 2020-06-20 at 09 29 04

### CODE REVIEW

File: lib/usecases/applicationDetails.js [hunk 1]
```diff
@@ -65,11 +65,17 @@ export default async ({ clientGeneratedId }) => {
         councilRentAccountNumber: application.council_rent_account_number,
         councilTaxNumber: application.council_tax_number,
         businessAddress: {
-          uprn: application.business_address_uprn,
+          uprn:
+            application.business_address_uprn === '0000000'
```
Comment (neilkidd) [inline lib/usecases/applicationDetails.js:69]: A bit of me dies inside when I see empty strings.
I wonder if we should leave it as it is? As finding the manual ones will need us to search for both '0000000' and empty string.
WDYT?

Comment (sirLisko) [reply lib/usecases/applicationDetails.js:69]: I know, the cleanest way would be having `uprn` and `postcode` not mandatory and remove the fake values we stored in the DB. This is a quick (hacky) win 😅

Comment (WillGibson) [reply lib/usecases/applicationDetails.js:69]: Instead of `''` maybe `'(Manually entered)'`?

Comment (WillGibson) [reply lib/usecases/applicationDetails.js:69]: Or `'(Entered manually)'`

Comment (sirLisko) [reply lib/usecases/applicationDetails.js:69]: @WillGibson the problem here is that we hardcode the value when we save it, it was a terrible hack made a couple of hours after going live 🙈

**Comment by sirLisko** [comment]
Merging, as a quick fix, we should address this in a better way but it's a bigger task, for the moment it's good enough.

**Pull request merged**